src/main/java/uk/gov/justice/probation/courtcaseservice/controller/mapper/CourtCaseResponseMapper.java

```diff
@@ -165,4 +134,27 @@ private static void addDefendantFields(CourtCaseResponseBuilder builder, Defenda
         // Offences
         builder.offences(mapOffencesFromDefendantOffences(defendantEntity.getOffences()));
     }
+
+    private static void addOffenderFields(CourtCaseResponseBuilder builder, DefendantEntity defendant, OffenderEntity offender) {
+        builder
+            .awaitingPsr(Optional.ofNullable(offender)
+                                    .map(OffenderEntity::getAwaitingPsr)
+                                    .orElse(defendant.getAwaitingPsr()))
```

Comment:
This fall back to defendant for offender based fields is only here to allow PACTs to continue to work. See https://dsdmoj.atlassian.net/browse/PIC-1907 which should be done as we remove the defendant fields